pkg/cmd/server/api/v1/conversions.go

```diff
@@ -131,6 +131,10 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.BindNetwork) == 0 {
 				obj.BindNetwork = "tcp4"
 			}
+			if obj.AllowRecursiveQueries == nil {
```

Comment:
In origin it doesn't bother me that much - someone can always turn off
recursive, but for now, being compatible and tested the same was as
upstream concerns me a great deal more.